crypto/rsa/rsa_sp800_56b_gen.c

```diff
@@ -195,11 +195,17 @@ static int rsa_validate_rng_strength(EVP_RAND_CTX *rng, int nbits)
 {
     if (rng == NULL)
         return 0;
+#ifdef FIPS_MODULE
+    /*
+     * This should become mainstream once similar tests are added to the other
```

Comment:
The comment looks like a TODO. :laughing:

Reply:
It isn't.  It's missing exactly four characters :)